tests/gradients/parameter_shift/test_parameter_shift_shot_vec.py

```diff
@@ -2109,7 +2109,10 @@ def test_trainable_coeffs(self, broadcast, tol):
 
         res = dev.execute([tape])
         expected = -c * np.sin(x) * np.sin(y) + np.cos(x) * (a + b * np.sin(y))
-        assert np.allclose(res, expected, atol=tol, rtol=0)
+        # This test is mostly for the shot vector pipeline check
+        # therefore only check a rough accuracy
+        assert qml.math.shape(res)[1] == qml.math.shape(shot_vec)[0]
+        assert np.allclose(res, expected, atol=0.002, rtol=0)
```

Comment:
```suggestion
        assert np.allclose(res, expected, atol=2*tol, rtol=0)
```
Or pylint won't allow